themes/custom_theme/valencia-theme/src/css/_custom.scss

```diff
@@ -0,0 +1,565 @@
+/* inject:imports */
+
+
+/* endinject */
+
+// Google Fonts
+@import url('https://fonts.googleapis.com/css?family=Roboto+Slab:400,700|Roboto:300,400,500,700');
+
+/* =============================================================
+>>> TABLE OF CONTENTS:
+================================================================
+# Sass Variables
+# Generic styles
+# Page Styling
+# Navigation
+# Typography
+# Buttons and links
+# Forms
+# Cards styling
+# Tables
+# Sections Styling
+# Footer
+# Responsive
+================================================================ */
```

Comment:
For your final, we do not need comment blocks such as this.  For development it is fine.